src/ZEO/__init__.py

```diff
@@ -22,10 +22,16 @@
 """
 
 def client(*args, **kw):
+    """
+    Shortcut for :class:`ZEO.ClientStorage.ClientStorage`.
+    """
     import ZEO.ClientStorage
     return ZEO.ClientStorage.ClientStorage(*args, **kw)
 
 def DB(*args, **kw):
+    """
+    Shortcut for :class:`ZODB.DB`.
```

Comment:
This might mention that it's using `client`.
```suggestion
    Shortcut for creating a :class:`ZODB.DB` using a ZEO :func:`client`.
```